luda-editor/client/src/pages/sequence_edit_page/loaded/update/mod.rs

```diff
@@ -173,6 +195,47 @@ impl LoadedSequenceEditorPage {
                         .unwrap();
                     });
                 }
+                &Event::CutTheCut { cut_id } => {
```

Comment:
Now I understand where the `the` come from😂

Reply:
CutCut